integration-guides/verizon.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Verizon Connect
+---
+
+To connect your Verizon telematics device integration with PathStack, you will need to be provided with a Verizon Connect App ID, a username and password for accessing the API.
+
+To find these credentials, please follow the steps below
+
+<Steps>
+  <Step title="Log into the Verizon Developer Portal">
+    To get an App ID from Verizon, you will need to log in or register for Verizon Connect’s Developer Portal. For further information and detailed instructions for signing up to the Verizon Developer Portal, see the Verizon Reveal help centre. 
+      <img src="images/Integration-guides/vzc_1.png" />
+
+  </Step>
+  <Step title="Request an APP ID">
+   Once you have access to the Developer portal, you will need to register for an App ID. For detailed instructions on how to do this, see the Verizon Reveal help guides. You will need to work with the team at Verizon to get the following details:
+         <Steps>
+              <Step title="App ID ">
+              </Step>
+              <Step title="Username">
+              </Step>
+              <Step title="Password">
+         </Step>
+</Steps>
+<img src="images/Integration-guides/vzc_2.png" />
+  </Step>
+</Steps>
+
+<Warning>For additional support and information, contact Verizon Connect at 1800 039 642 or [(supportaus@verizonconnect.com).](supportaus@verizonconnect.com)</Warning>
```

Comment:
Remove the link and just leave supportaus@verizonconnect.com.
e.g. 
Before:
`[(supportaus@verizonconnect.com).](supportaus@verizonconnect.com)`
After:
`supportaus@verizonconnect.com`
Do this for all emails. That way when you click the link it brings up your mail

Reply:
Also make sure that the email addresses are consistent, they all have brackets or none do